docs/ref/checks.txt

```diff
@@ -141,6 +141,8 @@ If you're using MySQL, the following checks will be performed:
   ``max_length`` > 255.
 * **mysql.W002**: MySQL Strict Mode is not set for database connection
   '<alias>'. See also :ref:`mysql-sql-mode`.
+* **mysql.W003**: MySQL does not support a database index on
```

Comment:
We have here a slightly different situation. Oracle database support a database index on a 
text fields (e.g. `CTXSYS.CONTEXT`, see [[1]](http://docs.oracle.com/database/121/CCREF/cdatadic.htm#CCREF0200)), but you need to use different operators to use them e.g. `CONTAINS` instead of `LIKE` etc., so a database backend doesn't support a database index on a text fields but a database itself supports it. But I think that you're right and we can handle these situations with a single warning:

***fields.W162**: ``<database>`` does not support a database index on ``<field data type>`` columns.*